data/abilities.js

```diff
@@ -147,20 +147,18 @@ exports.BattleAbilities = {
 		num: 107
 	},
 	"arenatrap": {
-		desc: "Prevents adjacent opposing Pokemon from choosing to switch out unless they are immune to trapping or have immunity to Ground.",
-		shortDesc: "Prevents adjacent foes from choosing to switch unless they are immune to Ground.",
+		desc: "Prevents adjacent opposing Pokemon from choosing to switch out unless they are immune to trapping or are ungrounded.",
+		shortDesc: "Prevents adjacent foes from choosing to switch unless they are ungrounded.",
```

Comment:
Good catch, but I'd prefer "airborne" to "ungrounded" since I've used it elsewhere before.